types/three/examples/jsm/helpers/ViewHelper.d.ts

```diff
@@ -0,0 +1,19 @@
+import {
+    Camera,
+	Object3D,
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/three/examples/jsm/helpers/ViewHelper.d.ts;
  Line: 3;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

Reply:
Updated to spacing.